tests/contrib/funders/test_funders_schema.py

```diff
@@ -10,29 +10,85 @@
 import pytest
 from marshmallow import ValidationError
 
-from invenio_vocabularies.contrib.funders.schema import FunderRelationSchema
+from invenio_vocabularies.contrib.funders.schema import FunderRelationSchema, \
+    FunderSchema
 
 
-def test_valid_full():
-    valid_full = {
+def test_valid_full(app, funder_full_data):
+    loaded = FunderSchema().load(funder_full_data)
+    assert funder_full_data == loaded
+
+
+def test_invalid_no_name():
+    invalid_no_name = {
+        "id": "10.5281/zenodo.9999999",
+    }
+    with pytest.raises(ValidationError):
+        data = FunderSchema().load(invalid_no_name)
+
+
+def test_invalid_empty():
+    invalid_empty = {}
+    with pytest.raises(ValidationError):
+        data = FunderSchema().load(invalid_empty)
+
+
+def test_invalid_empty_name():
+    invalid_empty_name = {
+        "id": "10.5281/zenodo.9999999",
+        "name": ""
+    }
+    with pytest.raises(ValidationError):
+        data = FunderSchema().load(invalid_empty_name)
+
+
+def test_invalid_id_format():
```

Comment:
very very minor: Im not sure if all these tests are needed, or are responsibilities of other modules... We can sit down and think about this on Monday.

Reply:
I have made some changes to this file, very minor:
- It is good to test fields that are defined here (e.g. `country`). However, others like `identifiers` that come from marshmallow-utils (our module, not 3rd party) we tend to trust and avoid testing many times.
- The relation schema was missing validation.